tests/support/gitfs.py

```diff
@@ -653,7 +653,9 @@ def tearDownClass(cls):
         for proc in (cls.case.nginx_proc, cls.case.uwsgi_proc):
             if proc is not None:
                 try:
-                    proc.send_signal(signal.SIGQUIT)
+                    proc.send_signal(signal.SIGTERM)
+                    if proc.is_running():
```

Comment:
Not too familiar with this code in particular, but I would consider adding a tiny sleep before the if here. Responding to SIGTERM is not going to be instant, even 10-100ms might do the trick. Othewise, I would just stick with one "aggressive" signal.

Secondly, SIGQUIT is perhaps not what you want. I think you might be more interested in doing SIGKILL.

https://en.wikipedia.org/wiki/Signal_(IPC)#SIGQUIT